tests/transports-governance/conftest.py

```
@@ -0,0 +1,476 @@
+            item.add_marker(pytest.mark.edge_case)
+
+
+class GovernanceTestClient:
+    """HTTP client for governance API testing with comprehensive error handling"""
+
+    def __init__(self, base_url: str = API_BASE):
+        self.base_url = base_url
+        self.session = requests.Session()
+        self.session.headers.update({"Content-Type": "application/json"})
+
+    def request(self, method: str, endpoint: str, **kwargs) -> requests.Response:
```

Ruff (0.12.2)

Missing type annotation for `**kwargs`

(ANN003)